samples/CommunityToolkit.Maui.Sample/Platforms/MacCatalyst/Info.plist

```diff
@@ -30,5 +30,7 @@
 	<string>ISpeechToText Uses Speech Recognition</string>
 	<key>NSMicrophoneUsageDescription</key>
 	<string>ISpeechToText Uses Microphone</string>
+	<key>NSCameraUsageDescription</key>
+	<string>ICameraView Uses Camera</string>
```

Comment:
Reminder that we'll need to add this to the `Platform specific initialization` section of the `CameraView` docs